src/test/PSMCallee.t.sol

```diff
@@ -89,28 +96,38 @@ contract PSMCalleeTest is DSTest {
     address usdp;
     address dai;
     address psm;
+    address spotter;
+
+    Vm public constant vm = Vm(hevm);
 
     function setUp() public {
-        clipper = Chainlog(chainlog).getAddress("MCD_CLIP_PAXUSD_A");
+        clipper = Chainlog(chainlog).getAddress("MCD_CLIP_USDC_A");
         address daiJoin = Chainlog(chainlog).getAddress("MCD_JOIN_DAI");
-        usdp = Chainlog(chainlog).getAddress("PAXUSD");
+        usdp = Chainlog(chainlog).getAddress("USDC");
         dai = Chainlog(chainlog).getAddress("MCD_DAI");
         callee = new PSMCallee(daiJoin);
         vat = Chainlog(chainlog).getAddress("MCD_VAT");
         Vat(vat).hope(clipper);
         tail = Clipper(clipper).tail();
-        psm = Chainlog(chainlog).getAddress("MCD_PSM_PAX_A");
+        psm = Chainlog(chainlog).getAddress("MCD_PSM_USDC_A");
+
+
+        // Emulate spell - set LR to 15000%
+        spotter = Chainlog(chainlog).getAddress("MCD_SPOT");
+        vm.prank(Chainlog(chainlog).getAddress("MCD_PAUSE_PROXY"));
+        Spotter(spotter).file("USDC-A", "mat", 150000 * RAY);
+        Spotter(spotter).poke("USDC-A");
```

Comment:
should this be done in the specific test?  maybe we move all the collateral related changes to their tests rather than setup? or have `_setupUSDC` and `_setupUSDP`?